backend/src/main/java/gov/cdc/usds/simplereport/service/PersonService.java

```diff
@@ -50,65 +62,127 @@ private void updatePersonFacility(Person person, UUID facilityId) {
     person.setFacility(facility);
   }
 
-  @AuthorizationConfiguration.RequirePermissionSearchPatients
-  public List<Person> getPatients(UUID facilityId, int pageOffset, int pageSize) {
-    return _repo
-        .findByFacilityAndOrganization(
-            _os.getFacilityInCurrentOrg(facilityId),
-            _os.getCurrentOrganization(),
-            false,
-            PageRequest.of(pageOffset, pageSize, NAME_SORT))
-        .toList();
+  // Specifications filters for queries
+  private Specification<Person> inCurrentOrganizationFilter() {
+    return (root, query, cb) ->
+        cb.equal(root.get(SpecField.ORGANIZATION), _os.getCurrentOrganization());
   }
 
-  @AuthorizationConfiguration.RequirePermissionSearchPatients
-  public long getPatientsCount(UUID facilityId) {
-    return _repo.countAllByFacilityAndOrganization(
-        _os.getFacilityInCurrentOrg(facilityId), _os.getCurrentOrganization(), false);
+  // Note: Patients with NULL facilityIds appear in ALL facilities.
+  private Specification<Person> inFacilityFilter(UUID facilityId) {
+    return (root, query, cb) ->
+        cb.and(
+            cb.equal(root.get(SpecField.ORGANIZATION), _os.getCurrentOrganization()),
```

Comment:
Why not take this out, since it's on both sides of an `if` where it's used?

Reply:
I was trying to error on the side of security (it should always include an Organization restrict).
[When we refactor Specifications, it should _always_ include an organization compare somewhere.]

Right now the calling code does this:
```
    Specification<Person> filter = isDeletedFilter(isArchived);
    if (facilityId == null) { // admin call to get all users
      filter = filter.and(inCurrentOrganizationFilter());
    } else {
      filter = filter.and(inFacilityFilter(facilityId));
    }
```
Do you mean for it to look something more like this?

```
    Specification<Person> filter = inCurrentOrganizationFilter();
    filter = filter.and(isDeletedFilter(isArchived));
    if (facilityId != null) { // admin call to get all users
      filter = filter.and(inFacilityFilter(facilityId));
    }
```
... which would allow the removal of the current organization check